RBAC phase one (#4236)
timescale/docs: use-timescale/security/read-only-role.md

---

> title: Role-based access to your data

Reviewer: This should be something along the lines of manage or administer service level data security

excerpt: Learn about the available user roles to access data in your database
products: [cloud]
keywords: [client credentials, accounts, users, members, read-only, security]
tags: [authentication, credentials, members, security]
---

> # Role-based access to your data

Reviewer: same as the title.

> When you create a $SERVICE_SHORT, $CLOUD_LONG assigns you the `tsdmadmin` role. This role has the full scope of permissions to modify data in your $SERVICE_SHORT, but it is not a superuser. $CLOUD_LONG does not provide superuser access. 

Reviewer: ```suggestion
When you create a $SERVICE_SHORT, $CLOUD_LONG assigns you the tsdmadmin role. This role has full permissions to modify data in your $SERVICE_SHORT. However,  $CLOUD_LONG does not provide superuser access. tsdmadmin is not a superuser. 
```

> As `tsdmadmin`, you can use standard $PG means to create other roles or assign individual permissions. This page explains how to create a read-only role for your database. Note that adding a read-only role does not provide resource isolation. If you want to restrict the access of a read-only user, as well as isolate resources, you can create a read replica instead. For more information, see [Read scaling][read-scaling].

Reviewer: ```suggestion
As tsdmadmin, you can use standard $PG means to create other roles or assign individual permissions. This page shows you how to create a read-only role for your database. Adding a read-only role does not provide resource isolation. To restrict the access of a read-only user, as well as isolate resources, create a [read replica][read-scaling] instead. 
```

<Highlight type="important">

> The user roles for managing data in the underlying database and administering $SERVICE_SHORTs in $CONSOLE_LONG do not overlap. This page describes the user roles on the database level. For user roles available in $CONSOLE_SHORT, see [Role-based access to Tiger Cloud projects][console-rbac].

Reviewer: I made a comment previously in the doc. Can you follow that please.
